<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>501</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="page_welcome" name="page_1"/>
    <widget class="page_rules" name="page_2"/>
    <widget class="page_twoplayers" name="page_3"/>
    <widget class="page_tournament" name="page_4"/>
    <widget class="page_allplayers" name="page_5"/>
   </widget>
   <widget class="QPushButton" name="welcomeButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rulesButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="twoplayerButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tournamentButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>520</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="allplayerButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>520</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>page_welcome</class>
   <extends>QWidget</extends>
   <header location="global">page_welcome.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>page_twoplayers</class>
   <extends>QWidget</extends>
   <header location="global">page_twoplayers.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>page_rules</class>
   <extends>QWidget</extends>
   <header location="global">page_rules.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>page_tournament</class>
   <extends>QWidget</extends>
   <header location="global">page_tournament.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>page_allplayers</class>
   <extends>QWidget</extends>
   <header location="global">page_allplayers.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
